aws/modules/infrastructure_modules/eks/variables.tf

```diff
@@ -139,3 +139,15 @@ variable "image_gc_low_threshold_percent" {
   type        = number
   default     = 80
 }
+
+variable "create_kms_key" {
+  description = "[ Warn: breaking-change ] Making this value false will allow passing a custom KMS key via the provider_key_arn configuration"
+  type        = bool
+  default     = true
+}
+
+variable "kms_key_administrators" {
```

Comment:
Is this variable used anywhere?

Reply:
Hi Jack
This is used in EKS Cluster Config
This enables to pass IAM Role to the policy which will be used for deployment where as in default scenario it falls back to the user used in IAC deployments.

Reply:
I see you've added the variable `` here but it's not being used anywhere in the PR?

Reply:
Yeah sorry will take it out, doesn't require to be defined here.